modules/service/controllers/globalrbac/handler.go

```diff
@@ -99,3 +170,11 @@ func (h *handler) generate(obj *riov1.Service, status riov1.ServiceStatus) ([]ru
 	}
 	return os.All(), status, nil
 }
+
+func serviceNameAndNamespace(obj runtime.Object) (serviceName string, serviceNamespace string, err error) {
```

Comment:
Nit: remove serviceName and serviceNamespace in return arguments?

Reply:
I want the function signature to provide information on what it is returning, it is a short function though